simba/schedule.py

```diff
@@ -604,6 +628,43 @@ def generate_scenario(self, args):
                             ext_load["csv_file"] = str(ext_load_path)
                             events["fixed_load"][gc_name + " ext. load"] = ext_load
 
+                        # temporary lowering of grid connector max power during peak load windows
+                        if time_windows is not None:
+                            grid_operator = station.get('grid_operator')
+                            if grid_operator is None:
+                                warnings.warn('Peak Load Windows given, '
+                                              f'but {gc_name} has no grid operator')
+                            # get relevant peak load windows
```

Comment:
every following condition expects grid_operator to be not None. Use break to make it clear following blocks can be discarded. Following conditions can be simplified accordingly.
The same thing for windows. e.g. if windows is None: break.

Reply:
In the electrified_station.json it says
`        "grid_operator" : "default_grid_operator", // optional: set grid operator for cost calculation. Default is "default_grid_operator"
`
So I would expect that if no grid operator is given "default_grid_operator" is aassigned to all other stations. How can it be none then?

Reply:
Resolved through change of default grid operator. A `break` would also skip the subsequent vehicle inits/event generations.